src/pages/MainPage/components/CarouselArea/index.tsx

```diff
@@ -0,0 +1,109 @@
+import { useEffect, useState } from 'react';
+import { css } from '@emotion/react';
+import useEmblaCarousel from 'embla-carousel-react';
+import { CaretLeft, CaretRight, HourGlass } from '@/assets/icons';
+import IconButton from '@/components/IconButton';
+import COLORS from '@/constants/colors';
+import Chip from '@/components/Chip';
+import textStyles from '@/styles/textStyles';
+import useLetterSlides from '../../hooks/useLetterSlides';
+import styles from './styles';
+import { BOTTLES_LETTER, BOTTLES_REPLY } from './bottleData';
+
+const CarouselArea = () => {
+  const [selectedIndex, setSelectedIndex] = useState(0);
+
+  const [emblaRef, emblaApi] = useEmblaCarousel({
+    loop: true,
+  });
+  const { slides } = useLetterSlides();
+
+  useEffect(() => {
+    if (!emblaApi) {
+      return;
+    }
+
+    emblaApi.on('select', () => {
+      setSelectedIndex(emblaApi.selectedScrollSnap());
+    });
+  }, [emblaApi]);
+
+  return (
+    <>
+      <section css={styles.viewport} ref={emblaRef}>
+        <div css={styles.container}>
+          {slides.map(({ id: slideId, letters, replies }) => (
+            <article key={slideId} css={styles.slide}>
+              {letters.map((letter, idx) => (
+                <div key={idx} css={BOTTLES_LETTER[idx]?.position}>
+                  <img
+                    src={BOTTLES_LETTER[idx]?.src}
+                    alt="물병"
+                    css={styles.bottleImage}
+                    onClick={() =>
+                      console.log(
+                        `TODO: ${letter.letterId} 번 편지 답장 보러가기`,
+                      )
+                    }
+                  />
+                  <Chip
+                    variant="bottle-tag"
+                    css={[styles.timeChip, BOTTLES_LETTER[idx]?.chipPosition]}
+                  >
+                    <HourGlass width="1rem" height="1rem" />
+                    <p css={textStyles.b4m}>00h</p>
+                  </Chip>
+                </div>
+              ))}
+
+              {replies.map((reply, idx) => (
+                <div key={idx} css={BOTTLES_REPLY[idx]?.position}>
+                  <img
+                    src={BOTTLES_REPLY[idx]?.src}
+                    alt="물병"
+                    css={styles.bottleImage}
+                    onClick={() =>
+                      console.log(
+                        `TODO: ${reply.letterId} 번 편지 답장 보러가기`,
+                      )
+                    }
+                  />
+                </div>
+              ))}
+            </article>
+          ))}
+        </div>
+      </section>
+
+      {slides.length > 1 ? (
+        <>
```

Comment:
p5;
큰 차이는 없지만 삼항연산자 보다는 이렇게 표현하는 게 더 좋을 것 같습니다!
```suggestion
{slides.length > 1 && (
  // 코드
)}
```

Reply:
조건부 렌더링에 단축 평가(&&) 를 사용했을 때 좌변의 타입이 boolean이 아니면 간혹 '0' 이 렌더링 되는 등의 문제를 겪었어서 삼항 연산자를 선호하긴 했는데.. 여기에선 명확한 boolean 형태라서 단축 평가 표현도 괜찮겠네요!! 👍😊

https://velog.io/@moonshadow/React%EC%97%90%EC%84%9C-%EC%A1%B0%EA%B1%B4%EB%B6%80-%EB%A0%8C%EB%8D%94%EB%A7%81%EC%97%90-%EB%A5%BC-%EC%93%B0%EC%A7%80-%EB%A7%90%EB%9D%BC%EA%B3%A0

Reply:
값이 `boolean`이 아닌 경우에는 **삼항 연산자**를 쓰는 게 좋군요 👍 저도 배워갑니다!